src/client/callers/agentLockAll.ts

```diff
@@ -0,0 +1,8 @@
+import {UnaryCaller} from "@matrixai/rpc";
+import {ClientRPCRequestParams, ClientRPCResponseResult} from "@/client/types";
+
+const agentLockAll = new UnaryCaller<
+  ClientRPCRequestParams,
+  ClientRPCResponseResult
+>();
```

Comment:
The agent callers does something like this for the types.

``` ts
type CallerTypes = HandlerTypes<NodesClaimsGetHandler>;

const nodesClaimsGet = new ServerCaller<
  CallerTypes['input'],
  CallerTypes['output']
>();
```

This needs to be replicated across all the callers as well.

Reply:
Could you elaborate on this?

Reply:
Look at how the `nodes/agent/callers` do this and do the same thing.

Reply:
Get the CallerTypes.